src/Common/src/CoreLib/System/Runtime/CompilerServices/AsyncIteratorMethodBuilder.cs

```diff
@@ -42,7 +42,7 @@ public struct AsyncIteratorMethodBuilder
         /// <param name="stateMachine">The state machine instance, passed by reference.</param>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void MoveNext<TStateMachine>(ref TStateMachine stateMachine) where TStateMachine : IAsyncStateMachine =>
-#if CORERT
+#if CORERT || MICROSOFT_BCL_ASYNCINTERFACES_PACKAGE
```

Comment:
Does `#if PROJECTN` above need to be changed to `#if PROJECTN || MICROSOFT_BCL_ASYNCINTERFACES_PACKAGE` as well?

Reply:
I'd thought about it and decided "no", that it's only necessary for debugging and debugging is likely already going to be subpar. But on further reflection, might as well give it the best hope. It'll require a few other tweaks. I'm just going to copy and modify the file.